docs/introduction.rst

```diff
@@ -82,27 +82,66 @@ approaches to this:
    output for the related images, we can deduce certain aspects of the
    model's behavior.
 
+
+.. note:: These descriptions apply only to *using the model after it has been created*; nothing is implied about how the model is constructed.
+
+.. note:: This distinction applies to both the *concept* and *implementation*. A model could support white box reasoning but lack the APIs to access the internal state; conversely, it might provide complete API access yet provide no guidance on how that state influences the output. **API access to the model does not automatically imply white box XAI.**
```

Comment:
Can notes take multi-line strings? I might suggest a once-sentence-per-line if possible for future diff sanity.

Reply:
fill-paragraph-or-region seems to indent correctly for multi-line notes; applied to both.